src/styles/blocks/_page.scss

```diff
@@ -0,0 +1,31 @@
+.page {
+  &__body {
+    scroll-behavior: smooth;
+    font-family: "Poppins", Helvetica, sans-serif;
+    margin: 0;
+    background-color: #1d1d1d;
```

Comment:
please make sure to use colors via variables, esp if you are using them more than 1 time
You already have variable `$navTextColor:#1d1d1d;`

Make sure to re-check other colors as well